docs/tutorial-create-pages.md

```diff
@@ -33,8 +33,10 @@ function HelloWorld(props) {
 module.exports = HelloWorld;
 ```
 
+> Use any text editor to make the file, such as [Microsoft Visual Studio Code](https://code.visualstudio.com/download) or [Komodo Edit](https://www.activestate.com/komodo-edit).
+
 2. Go to http://localhost:3000/hello-world and you should be able to see the new page.
-1. Change the text within the `<p>...</p>` to "I can write JSX here!". The browser should refresh automatically to reflect the changes.
+1. Change the text within the `<p>...</p>` to "I can write JSX here!" and save the file again. The browser should refresh automatically to reflect the change.
```

Comment:
The list number is off.

Reply:
Markdown syntax will continue from the previous number, so it's fine (click the preview to have a look).